File: algo-c-to-_/examples/isbn.lua
--
--	from src/isbn.c
--
--	a part of main		to	isISBN10
--

local M = require 'checkdigit'

local isISBN10 = M.isISBN10

do
	local s = "4871483517"
	print(("%s is %s."):format(s, isISBN10(s) and "valid" or "invalid"))
end

File: algo-c-to-_/examples/isbn_.py
#
#	from src/isbn.c
#
#	a part of main		to	isISBN10
#

from checkdigit import isISBN10

def p(s):
    print("{0} is {1}.".format(s, isISBN10(s) and "valid" or "invalid"))

p("4871483517")
